CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 2.13.0 (2018-10-23)
+- Updated the [skyux-builder-plugin-stache](https://github.com/blackbaud/skyux-builder-plugin-stache/blob/master/CHANGELOG.md#151-2018-08-02) to `1.7.0`.
+  - This update includes the new `skyux stache-update` command to update their SKY UX, SKY UX Builder, and Stache dependencies for them, deleting node_modules and npm installing.
+- Style changes to the `stache-table-of-contents` component. [#512](https://github.com/blackbaud/stache2/pull/512)
+  - TOC now sits to the right of the content on full size views, and above the content on mobile views.
+  - TOC now affixes to the top screen and follows users down the page as they scroll on full size views.
+  - TOC now gives a visual indication where the user is currently on the page by highlighting the current section being viewed on full size views.
```

Comment:
I'd change "TOC now gives" to "Updated TOC to provide." ... I'd change "where the user is currently on the page" to "of the current location." I'd also delete "being viewed." ... Also, hyphenate "full-size."